.github/workflows/quay.yml

```diff
@@ -0,0 +1,19 @@
+name: quay
+
+on: pull_request
+
+# on:
+#   push:
+#     branches:
+#       - master
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    env:
+      header: 'Content-Type:application/json'
+      payload: '{"commit":"${GITHUB_SHA}","ref":"${GITHUB_REF}","default_branch":"master"}'
+      url: '${{ secrets.QUAY_WEBHOOK_URL }}'
+    steps:
+    - name: update container on quay.io
+      run: curl -H "$header" -d "$payload "$url"
```

Comment:
This looks potentially problematic as you're double-quoting in the run call `-d "$payload"` but the actual payload variable contains both single and double-quotes. I would try and deflate this entire `jobs.push.env` into the run section:

```bash
run: |
    curl -H "Content-Type: application/json" -d ..."
```

Or just explicitly dump the output of those environment variables in the `run` section as well to make sure it's rendering correctly.

Reply:
I think this was the first thing I tried, but I'll keep messing with it